qutebrowser/config/configcommands.py

```diff
@@ -250,6 +250,60 @@ def config_unset(self, option, temp=False):
         with self._handle_config_error():
             self._config.unset(option, save_yaml=not temp)
 
+    @cmdutils.register(instance='config-commands')
+    @cmdutils.argument('option', completion=configmodel.list_option)
+    def config_add_list(self, option, value, temp=False):
+        """Append a value to a config option that is a list.
+
+        This appends an option to a config setting that is a list.
+
+        Args:
+            option: The name of the option.
+            value: The value to append to the end of the dictionary.
+            temp: Don't touch autoconfig.yml.
```

Comment:
I'd prefer "Set value temporarily until qutebrowser is closed." from `:config-set` as it's a bit less technical (but I know this is probably copied from `:config-unset`).